From 94e0e5b77fc121a94ba0a46fa1a92a2ab985afda Mon Sep 17 00:00:00 2001
From: Garrett Brown <themagnificentmrb@gmail.com>
Date: Fri, 24 Dec 2021 02:39:15 -0800
Subject: [PATCH] Fix build error on macOS

Error was:

  CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
  Please set them or make sure they are set and tested correctly in the CMake files:
  library_abs
      linked by target "rviz_rendering" in directory rviz_rendering
---
 rviz_assimp_vendor/rviz_assimp_vendor-extras.cmake.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rviz_assimp_vendor/rviz_assimp_vendor-extras.cmake.in b/rviz_assimp_vendor/rviz_assimp_vendor-extras.cmake.in
index 8e41fe66..929f659c 100644
--- a/rviz_assimp_vendor/rviz_assimp_vendor-extras.cmake.in
+++ b/rviz_assimp_vendor/rviz_assimp_vendor-extras.cmake.in
@@ -25,7 +25,7 @@ foreach(library IN LISTS ASSIMP_LIBRARIES)
   if(IS_ABSOLUTE "${library}")
     list(APPEND rviz_assimp_vendor_LIBRARIES "${library}")
   else()
-    find_library(library_abs ${library} PATHS "${ASSIMP_LIBRARY_DIRS}" NO_DEFAULT_PATH)
+    find_library(library_abs assimp)
     list(APPEND rviz_assimp_vendor_LIBRARIES "${library_abs}")
   endif()
 endforeach()
-- 
2.32.0 (Apple Git-132)

